### DISCUSSION TITLE
Wondering if anyone knows of an EPL scoreboard being available

### DISCUSSION THREAD

**Discussion by jeepdrvr71** [riffnshred/nhl-led-scoreboard#253] [General]
Does anyone know if an EPL scoreboard has been developed? I know there is an available API to pull from. I just don't have a lot of experience in this and don't know where to start.

If one hasn't been developed could someone point me to where I should begin with coding?

Thanks in advance for any help

**Comment by riffnshred** [comment]
I remember spoting one on reddit but I think the code was not published. It was not like mine at all tho. All it was doing is pull data of a live game and show the abbreviations and the score. 

What I recommand to anyone starting from scratch is, keep it simple. Start by trying to ready the data from the api using python and print the data in a terminal. Then play around with the rgbmatrix repo using the python binding and just display some text on screen. Every new thing you do, do a simple version of it to understand what can be done and how before you start emplementing it into your main project.
